### PR TITLE
Track core member activities (Invite, Create, Remove)

### PR DESCRIPTION
**Pull request by kewitz**

### CODE REVIEW

File: server/graphql/v2/mutation/MemberInvitationMutations.js [hunk 1]
```diff
@@ -148,6 +149,21 @@ const memberInvitationMutations = {
 
       if (args.accept) {
         await invitation.accept();
+        if ([MemberRoles.ACCOUNTANT, MemberRoles.ADMIN, MemberRoles.MEMBER].includes(invitation.role)) {
```
Comment (Betree) [inline server/graphql/v2/mutation/MemberInvitationMutations.js:152]: I think we can put that in `MemberInvitation.accept`, IMO this is part of the business logic and we'll want to record the activity regardless of where we're accepting the invitation from.

File: server/models/Activity.js [hunk 2]
```diff
@@ -51,7 +51,9 @@ function defineModel() {
 
       hooks: {
         afterCreate(activity) {
-          notify(activity); // intentionally no return statement, needs to be async
+          if (activity.data?.notify !== false) {
+            notify(activity); // intentionally no return statement, needs to be async
+          }
```
Comment (Betree) [inline server/models/Activity.js:56]: Great addition

**Review by Betree — COMMENTED** [review]
Great implementation. We should also make sure we cover the member invitations

**Comment by kewitz** [comment]
Rebased.

File: server/graphql/v2/mutation/MemberMutations.js [hunk 3]
```diff
@@ -161,6 +162,20 @@ const memberMutations = {
         throw new ValidationFailed(`Member ${memberAccount.slug} does not exist in Collective ${account.slug}`);
       }
 
+      if ([MemberRoles.ACCOUNTANT, MemberRoles.ADMIN, MemberRoles.MEMBER].includes(this.role)) {
```
Comment (Betree) [inline server/graphql/v2/mutation/MemberMutations.js:165]: It seems that we've broken this condition in the latest commits (needs to use `args`, not `this` here). I'm pushing a fix in https://github.com/opencollective/opencollective-api/pull/7598.

**Pull request merged**